Formula/zsh-syntax-highlighting.rb

```diff
@@ -1,7 +1,9 @@
 class ZshSyntaxHighlighting < Formula
   desc "Fish shell like syntax highlighting for zsh"
   homepage "https://github.com/zsh-users/zsh-syntax-highlighting"
-  url "https://github.com/zsh-users/zsh-syntax-highlighting/archive/0.4.1.tar.gz"
+  url "https://github.com/zsh-users/zsh-syntax-highlighting.git",
+    :tag => "0.4.1",
+    :revision => "c19ee583138ebab416b0d2efafbad7dc9f3f7c4f"
   sha256 "971b9c1e881a8d60442e40003c9e4c98bd1d5243a32fdf386a217c4cd7815197"
```

Comment:
`sha256` is redundant here, since you can't checksum a directory, which is what git clones create.